packages/communication-react/review/communication-react.api.md

```diff
@@ -1231,6 +1232,56 @@ export interface ErrorBarStrings {
 // @public
 export type ErrorType = keyof ErrorBarStrings;
 
+// @beta
+export interface FileMetaData {
+    extension: string;
+    name: string;
+    url: string;
+}
+
+// @beta
+export interface FileSharingOptions {
+    accept?: string;
+    multiple?: boolean;
+    uploadHandler: FileUploadHandler;
+}
+
+// @beta
+export class FileUpload {
+    constructor(file: File);
+    cancelUpload(): void;
+    completeUpload(metaData: FileMetaData): void;
+    extension(): string;
+    failUpload(message: string): void;
+    file: File;
+    // (undocumented)
+    isUploaded(): boolean;
+    metaData?: FileMetaData;
+    off(event: typeof UPLOAD_PROGRESSED_EVENT, listener: UploadProgressListener): void;
```

Comment:
Does the API documentation also list off the events that a class subscribes to?

Reply:
Yes